dev-client/src/util.ts

```diff
@@ -0,0 +1,9 @@
+export function formatNames(firstName: string, lastName?: string) {
+  let formatted = '';
+  if (lastName !== undefined) {
+    formatted += lastName;
+    formatted += ', ';
+  }
+  formatted += firstName;
+  return formatted;
+}
```

Comment:
We have similar code in AccountAvatar.js:

    const name = `${user.firstName} ${user.lastName}`;

And AppBar.js

    {user.firstName} {user.lastName}

Should we consolidate name formatting code?

Reply:
I think this format is slightly different, here the order is lastName, firstName. But it would make sense to group the other two.

Reply:
@david-code I would write this as 
```js
[lastName, firstName].filter(Boolean).join(", ")
```

Reply:
Oh yeah, that's nice. If you want to make a PR with that change, I will approve it.